packages/fxa-shared/package.json

```diff
@@ -2,63 +2,212 @@
+    },
+    "./nestjs/*": {
+      "import": "./dist/esm/packages/fxa-shared/nestjs/*.js",
+      "require": "./dist/cjs/packages/fxa-shared/nestjs/*.js"
+    },
+    "./oauth/*": {
+      "import": "./dist/esm/packages/fxa-shared/oauth/*.js",
+      "require": "./dist/cjs/packages/fxa-shared/oauth/*.js"
+    },
+    "./payments/*": {
+      "import": "./dist/esm/packages/fxa-shared/payments/*.js",
+      "require": "./dist/cjs/packages/fxa-shared/payments/*.js"
+    },
+    "./payments/configuration/*": {
+      "import": "./dist/esm/packages/fxa-shared/payments/configuration/*.js",
+      "require": "./dist/cjs/packages/fxa-shared/payments/configuration/*.js"
+    },
+    "./payments/iap/*": {
+      "import": "./dist/esm/packages/fxa-shared/payments/iap/*.js",
+      "require": "./dist/cjs/packages/fxa-shared/payments/iap/*.js"
+    },
+    "./payments/iap/apple-app-store/types": {
+      "import": "./dist/esm/packages/fxa-shared/payments/iap/apple-app-store/types/index.js",
+      "require": "./dist/cjs/packages/fxa-shared/payments/iap/apple-app-store/types/index.js"
+    },
+    "./payments/iap/google-play/types": {
+      "import": "./dist/esm/packages/fxa-shared/payments/iap/google-play/types/index.js",
+      "require": "./dist/cjs/packages/fxa-shared/payments/iap/google-play/types/index.js"
+    },
+    "./payments/stripe": {
+      "import": "./dist/esm/packages/fxa-shared/payments/stripe.js",
+      "require": "./dist/cjs/packages/fxa-shared/payments/stripe.js"
+    },
+    "./payments/stripe-firestore": {
+      "import": "./dist/esm/packages/fxa-shared/payments/stripe-firestore.js",
+      "require": "./dist/cjs/packages/fxa-shared/payments/stripe-firestore.js"
+    },
+    "./scripts/*": {
+      "import": "./dist/esm/packages/fxa-shared/scripts/*.js",
+      "require": "./dist/cjs/packages/fxa-shared/scripts/*.js"
+    },
+    "./sentry": {
+      "import": "./dist/esm/packages/fxa-shared/sentry/index.js",
+      "require": "./dist/cjs/packages/fxa-shared/sentry/index.js"
+    },
+    "./sentry/*": {
+      "import": "./dist/esm/packages/fxa-shared/sentry/*.js",
+      "require": "./dist/cjs/packages/fxa-shared/sentry/*.js"
+    },
+    "./speed-trap/*": {
+      "import": "./dist/esm/packages/fxa-shared/speed-trap/*.js",
+      "require": "./dist/cjs/packages/fxa-shared/speed-trap/*.js"
+    },
+    "./subscriptions/*": {
+      "import": "./dist/esm/packages/fxa-shared/subscriptions/*.js",
+      "require": "./dist/cjs/packages/fxa-shared/subscriptions/*.js"
+    },
+    "./tracing/*": {
+      "import": "./dist/esm/packages/fxa-shared/tracing/*.js",
+      "require": "./dist/cjs/packages/fxa-shared/tracing/*.js"
+    }
   },
   "scripts": {
     "postinstall": "yarn build || true",
-    "build": "tsc --build && yarn copy-assets && yarn copy-sql",
+    "build": "tsc --build && tsc --build tsconfig.cjs.json && echo '{ \"type\": \"module\" }' > ./dist/esm/packages/fxa-shared/db/package.json && yarn copy-assets && yarn copy-sql",
```

Comment:
The extra `package.json` bit is _specifically_ for the db patcher script.  Because the transpiled files have .js extensions not .mjs node think they are cjs; the package.json fixes that.  I scoped it into the /db dir since the db patcher script was the only thing with that issue (that I'm aware of).